src/stats/dendrogram.jl

```diff
@@ -0,0 +1,123 @@
+"""
+    dendrogram(x, y; kwargs...)
+
+Draw a [dendrogram](https://en.wikipedia.org/wiki/Dendrogram),
+with leaf nodes specified by `x` and `y` coordinates,
+and parent nodes identified by `merges`.
+
+# Arguments
+- `x`: x positions of leaf nodes
+- `y`: y positions of leaf nodes (default = 0)
+# Keywords
+- `merges`: specifies connections between nodes (see below)
+- `treestyle`: one of `:??`, `:??` 
+"""
```

Comment:
```suggestion

# Extended help

$(ATTRIBUTES)
"""
```

when you enter `??` at the REPL, it shows the default values of the attributes.

Reply:
Oh neat, was just putting that in as a placeholder while I think of what to name then.